<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>837</width>
    <height>570</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Novel Downloader</string>
  </property>
  <property name="windowIcon">
   <iconset resource="media.qrc">
    <normaloff>:/assert/icon/book.ico</normaloff>:/assert/icon/book.ico</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>URL</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="content_url"/>
      </item>
      <item>
       <widget class="QPushButton" name="get_contents_button">
        <property name="text">
         <string>Get contents</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Local Directory</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="local_directory"/>
      </item>
      <item>
       <widget class="QPushButton" name="select_local_direcotry_button">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>25</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="download_button">
        <property name="text">
         <string>Download</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pause_button">
        <property name="text">
         <string>Pause</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="2" column="0">
     <widget class="QTableView" name="novel_content_view">
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <attribute name="horizontalHeaderCascadingSectionResizes">
       <bool>true</bool>
      </attribute>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>837</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="action_exit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="action_about"/>
   </widget>
   <widget class="QMenu" name="menuPlugin">
    <property name="title">
     <string>Plugin</string>
    </property>
    <widget class="QMenu" name="menuwutuxs">
     <property name="title">
      <string>wutuxs</string>
     </property>
     <addaction name="action_wutuxs_open_website"/>
    </widget>
    <addaction name="menuwutuxs"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuPlugin"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_exit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="action_about">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="action_wutuxs_open_website">
   <property name="text">
    <string>Open website</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="media.qrc"/>
 </resources>
 <connections/>
</ui>
